manifests/g/GoLang/Go/1/18/1.18.4/GoLang.Go.1.18.locale.en-US.yaml

```diff
@@ -0,0 +1,29 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS1.7-2-5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: GoLang.Go.1.18
+PackageVersion: 1.18.4
+PackageLocale: en-US
+Publisher: GoLang
+PublisherUrl: https://go.dev/
+PublisherSupportUrl: https://go.dev/help/
+PrivacyUrl: https://policies.google.com/privacy
+Author: The Go Authors
+PackageName: Go 1.18
+PackageUrl: https://go.dev/
+License: BSD 3-Clause License
+LicenseUrl: https://github.com/golang/go/blob/master/LICENSE
+Copyright: Copyright (c) 2009 The Go Authors. All rights reserved.
+CopyrightUrl: https://go.dev/copyright
+ShortDescription: Go is an open source programming language that makes it easy to build simple, reliable, and efficient software.
+Description: The Go programming language is an open source project to make programmers more productive. Go is expressive, concise, clean, and efficient. Its concurrency mechanisms make it easy to write programs that get the most out of multicore and networked machines, while its novel type system enables flexible and modular program construction. Go compiles quickly to machine code yet has the convenience of garbage collection and the power of run-time reflection. It's a fast, statically typed, compiled language that feels like a dynamically typed, interpreted language.
+Moniker: go-1-18
```

Comment:
Should the latest version just have the moniker `go` or `golang` or something? I think `winget install golang` should do something. If someone else disagrees, I'm all ears.